helpers/dom.js

```diff
@@ -206,7 +206,7 @@ export function isComposedAncestor(ancestorNode, node) {
 	}) !== null;
 }
 
-export function isVisible(node) {
+export function isVisible(node, parentIsKnownVisible = false) {
```

Comment:
I might suggest renaming this to `checkAncestors` - I think it may be clearer, at least to me. 🤷‍♂️ Afterall, maybe the caller knows it is visible, maybe it doesn't, but for one reason or another it doesn't want this code to walk the ancestors. In fact, in the below usage, I don't think it actually knows the parent is visible... it looks like it is assuming that, at least at the time of the call.

I know we haven't done this in these helpers, but I might be inclined to begin using an `options` argument for arguments that aren't required. This would help the readability of the consumer and make the API less fragile as well.

```javascript
export function isVisible(node, options) {
	options = Object.assign({ checkAncestors: true }, options);
	...
}
```